lib/src/widgets/overlay.dart

```diff
@@ -692,53 +788,55 @@ class _TapTarget extends StatelessWidget {
         assert(color != null),
         super(key: key);
 
-  double opacity() {
+  double get opacity {
     switch (state) {
       case FeatureOverlayState.opening:
-        return const Interval(0.0, 0.3, curve: Curves.easeOut)
+        return const Interval(0, 0.3, curve: Curves.easeOut)
             .transform(transitionProgress);
       case FeatureOverlayState.activating:
       case FeatureOverlayState.dismissing:
-        return 1.0 -
-            const Interval(0.7, 1.0, curve: Curves.easeOut)
+        return 1 -
+            const Interval(0.7, 1, curve: Curves.easeOut)
                 .transform(transitionProgress);
-      default:
-        return 1.0;
+      case FeatureOverlayState.closed:
+        return 0;
+      case FeatureOverlayState.opened:
+        return 1;
     }
+    throw ArgumentError.value(state);
```

Comment:
This has two purposes: prevent such an error from slipping through in the future and also remove a warning because the analyzer does not realize that the `switch` statement is exhaustive.